src/platform/plugins/shared/dashboard/public/dashboard_api/performance/query_performance_tracking.ts

```diff
@@ -123,8 +131,52 @@ function reportPerformanceMetrics({
 }) {
   const duration =
     loadType === 'dashboardSubsequentLoad' ? timeToData : Math.max(timeToData, totalLoadTime);
+  const groupedPerformanceMarkers = getPerformanceTrackersGroupedById(
+    PERFORMANCE_TRACKER_TYPES.PANEL
+  );
 
-  const e = {
+  // `groupedPerformanceMarkers` is a map of performance markers grouped by id.
```

Comment:
Then, this comment can be simplified to just

"Calculate duration for each panel"